api/swagger/server/restapi/resource/swagger.yml

```diff
@@ -736,10 +736,17 @@ paths:
                   description: Plugin description.
                   type: string
                   x-nullable: false
+                home:
```

Comment:
Why do you need that? Why don't you use /thyra/plugin/{author-name}/{plugin-name} as home ?

Reply:
This si done to make things easier on the front part.